src/index.ts

```diff
@@ -23,6 +23,12 @@ export async function parse(input: string): Promise<any> {
   const posts = await parsePosts(base, authors);
   const presentations = await parsePresentations(base, authors);
 
+  const replacements =
```

Comment:
Nit: kinda weird variable naming here (i.e. what is pv?) This is just counting the number of replacements for posts/presentations, right? Just for debugging purposes?

Reply:
Yeah, this is just in here for debugging purposes.  When I searched `addjs` in the `export.xml`, it wasn't even close to the number of matches we were doing in processing.  This was my way of measuring that.

pv, cv is just previous and current value.  If there's a cool es6 way to do `posts.sum(it.addjsReplacements, 0)` w/o reduce or lodash, let me know.  Always looking for way to write more readable code.